scripts/functional-tests.sh

```diff
@@ -159,6 +159,24 @@ fi
 # download the latest version of the ilab
 ilab model download
 
+test_oci_model_download_with_vllm_backend(){
+    REGISTRY_AUTH_FILE=$HOME/auth.json ilab model download --repository docker://quay.io/ai-lab/models/granite-7b-lab --release latest
+
+    ilab model serve --model-path models/instructlab/granite-7b-lab/ &> serve.log &
+    PID_SERVE=$!
+
+    wait_for_server
+
+    expect -c '
+        set timeout 120
+        spawn ilab chat
+        expect ">>>"
+        send "this is a test! what are you? do not exceed ten words in your reply.\r"
+        send "exit\r"
+        expect eof
+    '
+}
```

Comment:
You can remove all that, as discussed offline vLLM is not supported in this test so we only want to validate that the command succeeded.

Reply:
removed